radio/src/gui/horus/menu_model_mixes.cpp

```diff
@@ -330,8 +338,32 @@ void displayMixLine(coord_t y, MixData *md)
   displayFlightModes(MIX_LINE_FM_POS, y, md->flightModes, 0);
 }
 
+void displayMixStatus(uint8_t channel)
+{
+  static const BitmapBuffer * mixer_bmp = BitmapBuffer::loadMaskOnBackground("mask_sbar_mixer.png", MENU_TITLE_COLOR, HEADER_BGCOLOR);
```

Comment:
`static` here would only make sense if you loaded bitmaps only one time. The way it is now it is only consuming global RAM.

In fact I think that the bitmaps should be loaded only once when user enters this screen and released once the user leaves the screen.